src/base/AsyncJob.cc

```diff
@@ -199,15 +197,14 @@ AsyncJob::ReportAllJobs(StoreEntry *e)
 {
     PackableStream os(*e);
     // this loop uses YAML syntax, but AsyncJob::status() still needs to be adjusted to use YAML
-    const char *indent = "    ";
+    const std::string indent("  ");
```

Comment:
Please do not use std::string in primary Squid code. We have discussed that before.

Reply:
moved to SBuf